cypress/integration/task-2.js

```diff
@@ -1,94 +1,39 @@
 describe('Task 2', function() {
 
-    it('T-2: Users should be able to log in with "User" role and see appropriate tabs', function () {
-
-        const user = "TestCon User 6"
-        const role = "User"
-
-        cy.login(user, role)
-
-        //"User" role asserts
-        cy.url().should('include', '/time-logging')
-        cy.get('.page__title').contains('Timesheets')
-        cy.get('.calendar').should('be.visible')
-        cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('TestCon User 6')
-        cy.get('.main-nav').find('li').should('have.length', 1)
-        cy.get('.main-nav').contains("Time Logging").should('have.css', 'color','rgb(64, 76, 237)')
+    it('T-2: All user roles can log in and see appropriate tabs', function () {
         
-    })
-
-    it('T-2: Users should be able to log in with "Team Lead" role and see appropriate tabs', function () {
-
+        let roles = ["User", "Team Lead", "Manager", "Accountant", "Admin"]
+        const nav_length = {
```

Comment:
You could merge those two properties into an array of objects with both properties, something like:
`let roles = [
{
    name: "User",
    length: 1
}, 
{
    name: "Team Lead",
    length: 2
},
{
    ...
}]`

Reply:
That's a very good idea, thanks! 🙏